plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/client/NuageVspApiClient.java

```diff
@@ -21,10 +21,10 @@
 
 public interface NuageVspApiClient {
 
-    public void login() throws Exception;
+    void login() throws Exception;
```

Comment:
This ``throws`` is too broad.  Please enumerate the expected checked exceptions to be thrown.